app/src/main/java/com/github/alfabravo2013/readyforexams/presentation/ToolbarFragment.kt

```diff
@@ -0,0 +1,24 @@
+package com.github.alfabravo2013.readyforexams.presentation
+
+import android.os.Bundle
+import android.view.View
+import androidx.fragment.app.Fragment
+import com.github.alfabravo2013.readyforexams.R
+import com.github.alfabravo2013.readyforexams.databinding.FragmentToolbarBinding
+
+class ToolbarFragment : Fragment(R.layout.fragment_toolbar) {
```

Comment:
Did not mean to do this. What I meant was an abstract BaseFragment that would have a method that would do activity?.setToolbar(title) or something. Each fragment of yours would extend BaseFragment() abstract class and all fragments would have access to setToolbar method. The toolbar can be created inside activity_main layout, above nav_host .